src/modules/worker/worker.service.spec.ts

```diff
@@ -0,0 +1,191 @@
+let callCount = 0;
+
+jest.mock('../../infra/y-redis/api.service.js', () => {
```

Comment:
Warum geht hier kein deep mock mit golevelup?
api.service ist doch "nur" ein injectable das sollte direkt gehen. 
Dann kann man in den jeweiligen Test schreiben was zurück kommen soll.